independent-projects/enforcer-rules/pom.xml

```diff
@@ -35,6 +35,8 @@
         <maven.compiler.source>11</maven.compiler.source>
         <maven.compiler.release>11</maven.compiler.release>
 
+        <!-- Keeping 3.0.0.M3 because 3.2.1 requires class changes -->
```

Comment:
This comment looks weird?

Reply:
That's intentional. I've updated the plugin but not the API because this project is used in the enforcer-plugin in other projects that may have not bumped to 3.2.1

Reply:
In short, running the plugin in 3.2.1 supports rules written in 3.0.0-M3 but 3.0.0-M3 does not support rules written in 3.2.1

Reply:
Now that I see that comment should have been in line 39, it's for `enforcer-api.version`